splice_machine/src/main/java/com/splicemachine/derby/stream/function/CloneFunction.java

```diff
@@ -32,4 +32,9 @@ public CloneFunction(OperationContext<Op> operationContext) {
     public ExecRow call(ExecRow o) throws Exception {
         return o.getClone();
     }
+
+    @Override
```

Comment:
I wonder why we don't have a function overwriting the super class's nativeTransformation(). The super class's nativeTransaformation() returns the dataset AS IS, but this function is supposed to perform a clone of a row. Do we end up without doing a clone if calling map(Clonefunction) from NativeSparkDataset?

Reply:
That's right, the clone operation is not needed when dealing with NativeSparkDatasets.

Reply:
Got it, thanks!